bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,22 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  object Owned {
```

Comment:
Those wouldn't work for APIs that need to take Bufs though, right?  You'd need to do `bytes.as[Owned].unwrap`, which is practically as bad as

```scala
val Buf.Owned(bytes) = buf
val buf = Buf.Owned(bytes)
```

I think @ianoc's idea for a Buf.Owned namespace is reasonable.